lib/modules/manager/github-actions/readme.md

```diff
@@ -1,7 +1,8 @@
 The `github-actions` manager extracts dependencies from GitHub Actions workflow and workflow template files.
 It can also be used for Gitea and Forgejo Actions workflows as such are compatible with GitHub Actions workflows.
 
-If you like to use digest pinning but want to follow the action version tag, you can use the following sample:
+If you like to use digest pinning but want to follow the action version tag, you can use the sample below.
+The version tag can be in the format `(prefix-)(v)1.0.0`, where `prefix` and `v` are optional and `1.0.0` is the version number.
```

Comment:
This sentence is pretty hard to understand, so best to add to the examples below to clarify.